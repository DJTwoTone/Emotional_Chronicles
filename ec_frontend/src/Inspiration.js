import React from 'react';

function Inspiration () {

    return (
        <div>
        Inspiration goes here.    
        </div>
            )
}

export default Inspiration;