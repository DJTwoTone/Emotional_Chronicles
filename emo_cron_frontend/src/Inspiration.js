import React from 'react';

function Inspiration () {


    //this need deigned
    //possibly an emotion cloud as the background
    //let logged in users 
    
    return (
        <div>
        Inspiration goes here.    
        </div>
            )
}

export default Inspiration;