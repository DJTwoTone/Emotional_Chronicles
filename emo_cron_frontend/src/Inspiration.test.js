import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Inspiration from './Inspiration';

it('should render', function() {
    render(
        <MemoryRouter>
            <Inspiration />
        </MemoryRouter>
    );
})

it('should match the snapshot', function() {
    const { asFragment } = render(
        <MemoryRouter>
            <Inspiration />
        </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
})