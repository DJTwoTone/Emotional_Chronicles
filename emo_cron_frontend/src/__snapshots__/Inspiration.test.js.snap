// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="justify-content-center m-2 container-fluid"
  >
    <div
      class="m-3 card"
    >
      <div
        class="card-body"
      >
        <div
          class="display-3 shadow card-header"
        >
          Something to inspire you...
        </div>
        <div
          class="my-2 list-group"
        />
        <button
          class="mb-4 btn btn-dark"
          type="button"
        >
          GET MORE INSPIRATION
        </button>
        <div
          class="card-footer"
        >
          <p
            class="card-text"
          >
            Share some inspiration...
          </p>
          <form
            class="form-inline"
          >
            <input
              class="col-10 form-control"
              name="inspText"
              type="text"
              value=""
            />
            <button
              class="ml-2 shadow btn btn-dark"
              type="submit"
            >
              SHARE
            </button>
          </form>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
